app/root.tsx

```diff
@@ -109,6 +127,20 @@ export function CatchBoundary() {
         <meta name="viewport" content="width=device-width,initial-scale=1" />
         <Meta />
         <Links />
+        <script
+          async
+          id="gtag-init"
+          dangerouslySetInnerHTML={{
+            __html: `
+                window.dataLayer = window.dataLayer || [];
+                function gtag(){dataLayer.push(gaTrackingId);}
```

Comment:
I think here we need to use what's default in the docs `function gtag(){dataLayer.push(arguments);}` (**arguments** instead of `gaTrackingId`)

Otherwise, I'm getting an error `Uncaught ReferenceError: gaTrackingId is not defined`, but maybe I'm doing something wrong 🤔